edx_name_affirmation/tasks.py

```diff
@@ -37,7 +38,11 @@ def idv_update_verified_name_task(self, attempt_id, user_id, name_affirmation_st
                 'status': name_affirmation_status
              }
              )
-    verified_names = VerifiedName.objects.filter(user__id=user_id, verified_name=photo_id_name).order_by('-created')
+    verified_names = VerifiedName.objects.filter(
```

Comment:
#NIT add a comment here to explain that when multiple VerifiedName records exists, there might be records we want to update that is not the latest created.